### PR TITLE
Cleans up some of the NLS strings

### PR DESCRIPTION
**Pull request by meg1502**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -9,7 +9,7 @@
         <version>3.12</version>
     </parent>
     <artifactId>sirius-web</artifactId>
-    <version>17.3</version>
+    <version>17.4</version>
```
Comment (andyHa) [inline pom.xml]: 18.0

File: pom.xml [hunk 2]
```diff
@@ -9,7 +9,7 @@
         <version>3.12</version>
```
Comment (andyHa) [inline pom.xml]: 3.13

Comment (andyHa) [reply pom.xml]: +kernel update

File: src/main/resources/web_de.properties [hunk 3]
```diff
@@ -1,24 +1,23 @@
 BasicController.404 = Seite nicht gefunden
-BasicController.changesSaved = Die \u00c4nderungen wurden gespeichert
+BasicController.changesSaved = Die \u00C4nderungen wurden gespeichert
```
Comment (andyHa) [inline src/main/resources/web_de.properties]: unicode senf

File: src/main/resources/web_en.properties [hunk 4]
```diff
@@ -38,32 +36,32 @@ ManagedTask.user = User
 ManagedTaskController.run = Run
 ManagedTaskController.scripting = System Scripting
 ManagedTaskExecution.completed = The execution has been completed.
-ManagedTaskExecution.scheduled = Execution planned, waiting for free thread...
+ManagedTaskExecution.scheduled = Execution planned, waiting for free thread\u2026
 ManagedTaskExecution.started = The execution was started.
 ManagedTaskExecution.waitingForForkedTasks = Wait for started background jobs.
 RythmConfig.jsDateFormat = mm/dd/yyyy
```
Comment (Tobsucht) [inline src/main/resources/web_en.properties]: are those still used?

Comment (meg1502) [reply src/main/resources/web_en.properties]: yes these are available as "dateFormat" / "timeformat" in the DefaultGlobalContext

**Pull request merged**